backend/src/main/java/com/google/sps/data/TimeHelper.java

```diff
@@ -1,42 +1,37 @@
 package com.google.sps.data;
 
-import java.util.Date;
-import java.util.Calendar;
-import java.time.LocalDateTime;
+import java.time.ZonedDateTime;
 
-/** Class used to set reference date/time for testing and convert input timeAvailableUntil to date timestamp */
+/**
+ * Class used to set reference date/time for testing and convert input timeAvailableUntil to date
+ * timestamp
+ */
 public final class TimeHelper {
```

Comment:
I tried getting rid of the TimeHelper class and moving getNewTimeToday() to FindMatchQueryTest, but I was running into a lot of issues with ZonedDateTime being referenced from a non-static context that I couldn't fix so I decided to keep TimeHelper.

Reply:
Sure, but keep in mind to fix this in future PRs, because when we need to add new features, we should stick with one class to expand, so it's better to get rid of TimeHelper in the long run.